src/modules/ethereum/types/IncomingTransaction.ts

```diff
@@ -1,22 +1,50 @@
-import type { BigNumberish } from 'ethers';
-import type { TransactionRequest } from '@ethersproject/abstract-provider';
+import type { BytesLike, TransactionRequest } from 'ethers';
 import type { types } from 'zksync-ethers';
 import type { PartiallyRequired } from 'src/shared/type-utils/PartiallyRequired';
 
-export type IncomingTransaction = Omit<
+type AddressStr = string;
```

Comment:
Do we need this?

Reply:
I'm not sure. What do you think?
Could be typed as 
```
type AddressStr = `0x${string}`
```